Gemfile.jruby-1.9.lock

```diff
@@ -132,8 +132,8 @@ GEM
     logstash-codec-graphite (2.0.2)
       logstash-codec-line
       logstash-core (>= 2.0.0.beta2, < 3.0.0)
-    logstash-codec-json (2.1.0)
-      logstash-core (>= 2.0.0.beta2, < 3.0.0)
+    logstash-codec-json (2.1.1)
+      logstash-core (>= 2.0.0, < 3.0.0)
```

Comment:
as long as a single `logstash-core (>= 2.0.0, < 3.0.0)` doesn't change how bundler picks prereleased gems (I believe it shouldn't), we're ok